packages/airnode-examples/package.json

```diff
@@ -5,7 +5,7 @@
   "private": false,
   "description": "A collection of examples showcasing the features of Airnode",
   "scripts": {
-    "build": "yarn run clean && hardhat compile",
+    "build": "yarn run clean && hardhat compile && yarn tsc --build tsconfig.json",
```

Comment:
Yes, this enforces that we actually fix the TS errors. Otherwise, they would only show up in the IDE. Alternatively, we could run `yarn tsc --build tsconfig.json` from the root (because it references all monorepo projects). But this way feels more natural.